deps/v8/gypfiles/v8.gyp

```diff
@@ -2564,102 +2631,70 @@
+        "../src/torque/earley-parser.h",
+        "../src/torque/file-visitor.cc",
+        "../src/torque/file-visitor.h",
+        "../src/torque/global-context.h",
+        "../src/torque/implementation-visitor.cc",
+        "../src/torque/implementation-visitor.h",
+        "../src/torque/scope.cc",
+        "../src/torque/scope.h",
+        "../src/torque/source-positions.cc",
+        "../src/torque/source-positions.h",
+        "../src/torque/torque-parser.cc",
+        "../src/torque/torque-parser.h",
+        "../src/torque/type-oracle.cc",
+        "../src/torque/type-oracle.h",
+        "../src/torque/types.cc",
+        "../src/torque/types.h",
+        "../src/torque/utils.cc",
+        "../src/torque/utils.h",
       ],
     }, # torque_base
     {
-      'target_name': 'v8_torque',
-      'type': 'none',
+      'target_name': 'torque',
+      'type': 'executable',
       'toolsets': ['host'],
-      'dependencies': ['torque#host'],
-      'direct_dependent_settings': {
-        'include_dirs+': ['<(SHARED_INTERMEDIATE_DIR)'],
-      },
-      'actions': [
-        {
-          'action_name': 'run_torque',
-          'inputs': [  # Order matters.
-            '<(PRODUCT_DIR)/<(EXECUTABLE_PREFIX)torque<(EXECUTABLE_SUFFIX)',
-            '../src/builtins/base.tq',
-            '../src/builtins/array.tq',
-            '../src/builtins/array-copywithin.tq',
-            '../src/builtins/array-foreach.tq',
-            '../src/builtins/array-lastindexof.tq',
-            '../src/builtins/array-reverse.tq',
-            '../src/builtins/array-splice.tq',
-            '../src/builtins/array-unshift.tq',
-            '../src/builtins/typed-array.tq',
-            '../src/builtins/data-view.tq',
-            '../third_party/v8/builtins/array-sort.tq',
-          ],
-          'outputs': [
-            '<(SHARED_INTERMEDIATE_DIR)/torque-generated/builtin-definitions-from-dsl.h',
-            '<(SHARED_INTERMEDIATE_DIR)/torque-generated/builtins-array-from-dsl-gen.cc',
-            '<(SHARED_INTERMEDIATE_DIR)/torque-generated/builtins-array-from-dsl-gen.h',
-            '<(SHARED_INTERMEDIATE_DIR)/torque-generated/builtins-base-from-dsl-gen.cc',
-            '<(SHARED_INTERMEDIATE_DIR)/torque-generated/builtins-base-from-dsl-gen.h',
-            '<(SHARED_INTERMEDIATE_DIR)/torque-generated/builtins-typed-array-from-dsl-gen.cc',
-            '<(SHARED_INTERMEDIATE_DIR)/torque-generated/builtins-typed-array-from-dsl-gen.h',
-            '<(SHARED_INTERMEDIATE_DIR)/torque-generated/builtins-data-view-from-dsl-gen.cc',
-            '<(SHARED_INTERMEDIATE_DIR)/torque-generated/builtins-data-view-from-dsl-gen.h',
-          ],
-          'action': ['<@(_inputs)', '-o', '<(SHARED_INTERMEDIATE_DIR)/torque-generated'],
+      'dependencies': ['torque_base'],
+      'defines!': [
+        '_HAS_EXCEPTIONS=0',
+        'BUILDING_V8_SHARED=1',
+      ],
+      # This is defined trough `configurations` for GYP+ninja compatibility
```

Comment:
`through`?

Reply:
👍 but even better, the comment is now irrelevant.